docs/display_panels.rst

```diff
@@ -2,147 +2,103 @@
 
 .. _display-panels:
 
+**************
 Display Panels
-==============
+**************
+The center of the window is called the workspace and it is where data is visualized. The workspace consists of display panels which display a piece of data (also called a data item). 
 
-Display Panels
---------------
-Display panels are used to show display items.
-
-You can drag items from the data panel into a display panel.
-
-You can put keyboard focus on a display panel by clicking on it.
-
-The display item with keyboard focus also serves as the primary selected item. Primary selections are indicated by a solid focus ring.
+.. image:: graphics/workspace_layout.png
+    :width: 937
+    :alt: Workspace Layout
 
-You can add secondary selected items by :kbd:`control` clicking (Windows/Linux) or :kbd:`command` (macOS) clicking on display panels. Secondary selections are indicated by a dotted focus ring.
+In a new project, the workspace will be filled by a single display panel. Use the toolbar to split the workspace into multiple smaller display panels, zoom into images, add graphics, and move an image.
 
-Processing and other commands apply to the primary selected items. Processing commands that need multiple inputs may use secondary selected items.
+To display a data item, drag it from the data panel into a display panel. You can also drag data files into a display panel directly from your desktop and from Explorer on Windows or Finder on macOS. Once the data item is in the display panel, you can use the utility panels surrounding the workspace to perform various functions, processes, manipulations, and calibrations to the selected image or line plot.
```

Comment:
I don't think this captures the behavior accurately. The display panel itself has its own functionality - for instance you can change the view, add graphics, make selections (line plot), etc. The utility panels allow you to perform additional operations.

Also, the wording of "to perform various functions, processes..." etc. could be streamlined. "the utility panels allow you to process, analyze, and edit the details of the data items in the display panels." or something like that.

Also, I'd like to stick to common terminology of "process", "analyze", "calibrate", "annotate" if possible.

Reply:
I'm not sure if this is the right place to mention the other functionality of display panels. I mention splitting and zooming panels in the workspace in the paragraphs above before any data is displayed, and I mention adding graphics and making selections in the paragraphs below after the data is displayed.

Reply:
How about just leaving out the sentence that starts with "Once the data item is in the display panel..."? At the very least, minimizing it with just "Many of the utility panels operate on the selected display panel."

Reply:
Check out my next iteration. I may have found a nice middle ground.